README.md

```diff
@@ -6,37 +6,118 @@
+In both cases, it only takes three very small and simple steps to add the DICOM Viewer Web Extension to the [`ocis_full` deployment example](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_full/) of your own installation:
+
+1. Navigate to the `/opt/compose/ocis/ocis_full` folder of your installation and copy [`dicom-viewer.yml`](https://github.com/owncloud/web-app-dicom-viewer/blob/main/dicom-viewer.yml) into the [`web_extensions`](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_full/web_extensions) subfolder.
+
+2. Add `DICOMVIEWER=:web_extensions/dicom-viewer.yml` to the `## oCIS Web Extensions ##` section of the `.env` file of your installation (file is located in `/opt/compose/ocis/ocis_full`).\
+Your `.env` file should now look like this:
+   ```
+   ## oCIS Web Extensions ##
+   # It is possible to use the oCIS Web Extensions to add custom functionality to the oCIS frontend.
+   # For more details see https://github.com/owncloud/web-extensions/blob/main/README.md
+   
+   <list of all web extensions>
+   
+   DICOMVIEWER=:web_extensions/dicom-viewer.yml
+   ```
+
+3. Append `${DICOMVIEWER:-}` to the `COMPOSE_FILE` variable at the very end of the last line of the `.env` file. This variable combines the configs of all the components that need to be loaded.
+
+   `COMPOSE_FILE=docker-compose.yml${OCIS:-} ... <variables of lots of other configs that are added to docker compose> ... ${DICOMVIEWER:-}`
+
+   After appending `${DICOMVIEWER:-}`, your `.env` file should look like this:
+   ```
+   ## IMPORTANT ##
+   # This MUST be the last line as it assembles the supplemental compose files to be used.
+   # ALL supplemental configs must be added here, whether commented or not.
+   # Each var must either be empty or contain :path/file.yml
+   COMPOSE_FILE=docker-compose.yml${OCIS:-}${TIKA:-}${S3NG:-}${S3NG_MINIO:-}${COLLABORA:-}${MONITORING:-}${IMPORTER:-}${CLAMAV:-}${ONLYOFFICE:-}${INBUCKET:-}${EXTENSIONS:-}${UNZIP:-}${DRAWIO:-}${JSONVIEWER:-}${PROGRESSBARS:-}${EXTERNALSITES:-}${DICOMVIEWER:-}
+   ```
+
+Done! Have fun using the [functionalities of the DICOM Viewer web extension](#functionalities-of-dicom-viewer-web-extension) on your installation!
+
+
+### Manual App Installation
+
+1. Download the zip file from the [releases page](https://github.com/owncloud/web-app-dicom-viewer/releases).
+2. Extract the zip file to the `apps` directory of your oCIS server. The `apps` directory is set using the `WEB_ASSET_APPS_PATH` environment variable.
+
+#### Prerequisites
+
+- Supported oCIS and Web Versions: oCIS (>= 6.2.x), Web (>= 9.x.x)
```

Comment:
@Salipa-Gurung in the section on build and run for development below we mention different approaches for loading the extension depending if it's oCIS 5.0.0 or >= 5.1.0 which sounds to me like the app runs even on version 5.0.0. 

Is there a difference in which versions are supported for deployment and development?

Reply:
During development we can make app work on oCIS 5.0.9 and >=5.1 as well but for deployment we kept minimum oCIS version `6.2.0` https://github.com/owncloud/awesome-ocis/blob/0138d4fd966fed78658e813eba6929fd4e4496aa/webApps/apps.json#L379C1-L379C30.

Reply:
Thanks for the info!